scripts/travis/package-deploy.sh

```diff
@@ -34,7 +34,7 @@ function package {
     local PLATFORM=$1
     local FILE_EXTENSION=$2
     # script start
-    if [ "$PLATFORM" == "linux-s390x" ]; then
+    if [ "$PLATFORM" == "linux-s390x" ] || [ "$PLATFORM" == "linux-arm64" ]; then
```

Comment:
I think we may want to follow suit and have dir names to also include the arch, to be consistent.

Reply:
Done.